### PR TITLE
Introduce test_07_combined_continue_gets_before_the_creation_of_accounts

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**
Execution a contract by eth key pair at the first time give the following [response](https://buildkite.com/cyberway/evm-loader/builds/1466#090cc57c-6fa9-4176-a22e-a0fa2dac5543/35-2628)
```javascript
{
  'code': -32002,
  'message': 'Transaction simulation failed: Error processing Instruction 1: invalid program argument',
  'data': {
    'accounts': None,
    'err': {
      'InstructionError': [
        1,
        'InvalidArgument'
      ]
    },
    'logs': [
      'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]',
      'Program log: src/storage_account.rs:65',
      'Program log: src/account_storage.rs:156 : Caller could not be Solana user. It must be neon-evm owned account',
      'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 26875 of 200000000 compute units',
      'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument'
    ]
  }
}
```

### CODE REVIEW

File: evm_loader/test_transaction.py [hunk 1]
```diff
@@ -76,6 +90,17 @@ def get_tx(self):
             'chainId': 111
         }
 
+    def get_tx_2(self):
```
Comment (s-medvedev) [inline evm_loader/test_transaction.py:93]: Please, no need to create unnecessary entities by copying. Reuse already exists code.

File: evm_loader/test_transaction.py [hunk 2]
```diff
@@ -88,6 +113,18 @@ def get_keccak_instruction_and_trx_data(self, data_start):
                                                     )
         return keccak_instruction, trx_data, sign
 
+    def get_keccak_instruction_and_trx_data_2(self, data_start):
```
Comment (s-medvedev) [inline evm_loader/test_transaction.py:116]: Adapt and reuse `get_keccak_instruction_and_trx_data`

File: evm_loader/test_transaction.py [hunk 3]
```diff
@@ -118,9 +155,42 @@ def get_account_metas_for_instr_05(self):
             AccountMeta(pubkey=TOKEN_PROGRAM_ID, is_signer=False, is_writable=False),
         ]
 
+    def get_account_metas_for_instr_05_2(self):
```
Comment (s-medvedev) [inline evm_loader/test_transaction.py:158]: Here too.

File: evm_loader/test_transaction.py [hunk 4]
```diff
@@ -52,6 +52,20 @@ def setUpClass(cls):
         cls.collateral_pool_address = create_collateral_pool_address(collateral_pool_index)
         cls.collateral_pool_index_buf = collateral_pool_index.to_bytes(4, 'little')
 
+        wallet_2 = RandomAccount()
+        cls.loader_2 = EvmLoader(wallet_2, evm_loader_id)
```
Comment (s-medvedev) [inline evm_loader/test_transaction.py:56]: Why do you need a second EvmLoader?

**Review by s-medvedev — CHANGES_REQUESTED** [review]

File: evm_loader/test_transaction.py (hunk 1 above)
Comment (vasiliy-zaznobin) [reply evm_loader/test_transaction.py]: Fixed

File: evm_loader/test_transaction.py (hunk 2 above)
Comment (vasiliy-zaznobin) [reply evm_loader/test_transaction.py]: Fixed

File: evm_loader/test_transaction.py (hunk 3 above)
Comment (vasiliy-zaznobin) [reply evm_loader/test_transaction.py]: Fixed

File: evm_loader/test_transaction.py (hunk 4 above)
Comment (vasiliy-zaznobin) [reply evm_loader/test_transaction.py]: Fixed

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**